pybliometrics/scopus/subject_classifications.py

```diff
@@ -89,7 +89,9 @@ def __init__(self,
         self._refresh = refresh
         self._query = str(query)
         self._view = ''
-        Search.__init__(self, query=query, api='SubjectClassifications', **kwds)
+        # Determine module
+        self._api = 'SubjectClassifications' if 'scopus' in self.__module__ else 'ScienceDirectSubjectClassifications'
+        Search.__init__(self, query=query, api=self._api, **kwds)
```

Comment:
If the api is passed as a parameter, it is not needed as underscore property. If it is needed elsewhere, it can stay underscore property but need not be passed as parameter.

So, is it used somewhere else? If not, let's just make pass a simple parameter.